src/components/Navigation/Navbar.js

```diff
@@ -1,75 +1,78 @@
 import React, { useState } from 'react';
-import { Link } from 'react-router-dom';
+import { NavLink } from 'react-router-dom';
 import './Navbar.css';
+import logo from '../../assets/images/F2P-logo.svg';
 
-function Navbar() {
+export default function Navbar() {
   const [click, setClick] = useState(false);
   const handleClick = () => setClick(!click);
   const closeMobileMenu = () => setClick(false);
   return (
     <>
       <nav className="navbar">
         <div className="navbar-container">
-          <Link to="/" className="navbar-logo" onClick={closeMobileMenu}>
-            Farm2People
-            <i className="fas fa-tractor" />
-          </Link>
+          <NavLink to="/" onClick={closeMobileMenu}>
+            <img src={logo} alt="Logo" className="navbar-logo" />
+          </NavLink>
           {/* eslint-disable-next-line */}
```

Comment:
Can the code be adjusted to not require the eslint disabling, or is this necessary?